concert/app/test/__init__.py

```diff
@@ -4,3 +4,4 @@
 os.environ["AWS_DEFAULT_REGION"] = "us-east-1"
 os.environ["EVENT_NAME"] = "CONCERT_LAUSANNE_2019"
 os.environ["DYNAMODB_TABLE_NAME"] = "testing-enchanted-brain"
+os.environ["API_URL"] = "https://abc.execute-api.us-east-1.amazonaws.com/Prod"
```

Comment:
Take a look at my PR to see how you can setup test-specific initialization by using directories.

Reply:
Done